examples/acmetool/responses

```diff
@@ -0,0 +1,15 @@
+# https://hlandau.github.io/acme/userguide#response-files
+#
+# This is a example of a response file, used with --response-file.
+# It automatically answers prompts for unattended operation.
+# grep for UniqueID in the source code for prompt names.
+# Pass --response-file to all invocations, not just quickstart.
+# If you don't pass --response-file, it will be looked for at "(state-dir)/conf/responses".
+# You will typically want to use --response-file with --stdio or --batch.
+# For dialogs not requiring a response, but merely acknowledgement, specify true.
+# This file is YAML. Note that JSON is a subset of YAML.
+"acme-enter-email": "hostmaster@example.com"
+"acme-agreement:https://letsencrypt.org/documents/LE-SA-v1.1.1-August-1-2016.pdf": true
+"acmetool-quickstart-choose-server": https://acme-staging.api.letsencrypt.org/directory
+"acmetool-quickstart-choose-method": proxy
+"acmetool-quickstart-install-cronjob": true
```

Comment:
We will also need to use ENV replacement here on `acme-staging.api` to `acme-v01.api`.